code/network/multi_endgame.cpp

```diff
@@ -308,6 +308,11 @@ int multi_quit_game(int prompt, int notify_code, int err_code, int wsa_error)
 		}
 	}		
 
+	// Log out of PXO, if needed
+	if (!(Game_mode & GM_STANDALONE_SERVER) && (Net_player->flags & NETINFO_FLAG_MT_CONNECTED)) {
+		multi_fs_tracker_logout();
+	}
```

Comment:
This really shouldn't be needed. Once a game has fully quit it will move to `multi_endgame_cleanup()` where the game will be logged out of the tracker. If there is a condition whereby that doesn't happen, and this change is required, then it's a bug.